sklearn/tests/test_base.py

```diff
@@ -509,3 +516,64 @@ def test_regressormixin_score_multioutput():
            "built-in scorer 'r2' uses "
            "multioutput='uniform_average').")
     assert_warns_message(FutureWarning, msg, reg.score, X, y)
+
+
```

Comment:
Please test that passing a kw that is not known by check_array raises a TypeError